### PR TITLE
Add Ember + Storybook writeup

### PR DESCRIPTION
**Pull request by amyrlam**
I also updated the template for Substack, can further update after the next issue.

@gossi sorry this didn't get in last issue, appreciate you sharing the link on GitHub and Discord! Please let me know if you have any feedback on the writeup! I haven't used Storybook much, but am excited to explore it more.

### CODE REVIEW

File: content/the-ember-times-issue-172.md [hunk 1]
```diff
@@ -23,16 +26,19 @@ The [first video](https://www.youtube.com/watch?v=G7QWvhMXMSc) goes through sett
...
+
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## [Storybook for Ember tutorial 🎨](https://www.learnstorybook.com/intro-to-storybook/ember/en/get-started/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Storybook](https://storybook.js.org/) has taken the world of **design systems** by storm! What is Storybook? Storybook makes building UI components isolated from the business logic and context of your app easier, sort of a "style guide as a service". Check out the new [Storybook for Ember tutorial](https://www.learnstorybook.com/intro-to-storybook/ember/en/get-started/) for guidance on setting up Storybook in your Ember app.
```
Comment (ijlee2) [inline content/the-ember-times-issue-172.md]: I thought the 2nd sentence was initially hard to understand (in particular, what "isolated" referred to). Maybe break it into two sentences?

```suggestion
[Storybook](https://storybook.js.org/) has taken the world of **design systems** by storm! What is Storybook? Storybook helps you build UI components that are isolated from business logic and context of your app. Sort of a "style guide as a service." Check out the new [Storybook for Ember tutorial](https://www.learnstorybook.com/intro-to-storybook/ember/en/get-started/) for guidance on setting up Storybook in your Ember app.
```

File: content/the-ember-times-issue-172.md [hunk 2]
```diff
@@ -23,16 +26,19 @@ The [first video](https://www.youtube.com/watch?v=G7QWvhMXMSc) goes through sett
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[Thomas Gossmann (@gossi)](https://github.com/gossi) spearheaded the Ember + Storybook effort and wrote a complementary blog post, [Ember with Storybook – Behind the Scenes](https://gos.si/blog/ember-with-storybook-behind-the-scenes/). The post touches on how Storybook and Ember are independent build pipelines, options for writing Storybook docs in Markdown, and different strategies for Storybook setup depending on your app architecture (e.g. monorepo vs. multirepo). But wait, there's more - you can use @gossi's open source Ember design system, hokulea ([repo](https://github.com/hokulea/hokulea) and [Storybook](https://hokulea.netlify.app/), as a reference!
```
Comment (ijlee2) [inline content/the-ember-times-issue-172.md]: I think we can create the effect of a short paragraph by showing a list of items:

```suggestion
[Thomas Gossmann (@gossi)](https://github.com/gossi) spearheaded the Ember + Storybook effort and wrote a complementary blog post, [Ember with Storybook – Behind the Scenes](https://gos.si/blog/ember-with-storybook-behind-the-scenes/). The post touches on a few ideas:

- How Storybook and Ember are independent build pipelines
- Options for writing Storybook docs in Markdown
- Different strategies for Storybook setup depending on your app architecture (e.g. monorepo vs. multirepo).

But wait, there's more - you can use @gossi's open source Ember design system, hokulea, as a reference!

- [GitHub Repo](https://github.com/hokulea/hokulea)
- [Storybook](https://hokulea.netlify.app/)
```

**Review by ijlee2 — APPROVED** [review]
🎉🎉🎉

**Comment by amyrlam** [comment]
thanks @ijlee2 @gossi! we can make edits upstream if needed as well!

also i will get the new email provider substack draft started, since we can have richer content than the blog

**Comment by gossi** [comment]
Thanks @amyrlam and @ijlee2. I fine with both of what you wrote. More than I asked for, super happy ☺️

**Pull request merged**